core/src/main/java/org/stellar/anchor/sep31/Sep31TransactionStore.java

```diff
@@ -18,28 +18,28 @@ public interface Sep31TransactionStore {
    *
    * @param transactionId The transaction ID.
    * @return The transaction document. null if not found.
-   * @throws SepException if error happens.
+   * @throws RuntimeException if error happens.
    */
-  Sep31Transaction findByTransactionId(String transactionId) throws SepException;
+  Sep31Transaction findByTransactionId(String transactionId) throws RuntimeException;
```

Comment:
My reason for changing it to RuntimeException is because findByTransactionId uses a CrudRepository method that can throw `IllegalArgumentException`, which extends RuntimeException.

We're not throwing any other exception on purpose for now, so I don't think it's currently possible to get an `AnchorException`.

What do you think of updating it to `throws RuntimeException, AnchorException`?

Reply:
In Java, the function does not need to declare `throws RuntimeException` and can be skipped.

The concern is from `Sep31TransactionStore` being an interface and we do not know what the implementation may throw. Therefore, giving a generic `throws AnchorException` will give the implementation to report an error wrapped in a sub-class of AnchorException and the caller to handle it in a uniform way.

Reply:
Fixed in https://github.com/stellar/java-stellar-anchor-sdk/pull/200/commits/3275a983d33848e436434326d4ed32433242563e.